### PR TITLE
change selector for profile links

### PR DESCRIPTION
**Pull request by rajkumaar23**
## Description

**Fixes** #20 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots of the change
image

### CODE REVIEW

**Comment by rajkumaar23** [comment]
@ayushjainrksh This selector works for me. If the same HTML is what you see, this should work for you too!

**Comment by ayushjainrksh** [comment]
@rajkumaar23 Thanks for the PR. I'm gonna try it out tomorrow. I want to know if adding the solution I suggested worked for you or changing the selector was the solution? You seem to have added both and I'm curious about what helped to solve the problem.

**Comment by rajkumaar23** [comment]
@ayushjainrksh No, adding what you suggested didn’t help. Hence, I changed the selectors. 
Still, retained your suggestion in case it prevents  breaking things in slow internet.

**Comment by ayushjainrksh** [comment]
I tried out the changes. These selectors are not present in my LinkedIn HTML. This is really strange as I thought we have the same HTML but turns out there's a difference.

The class `reusable-search__entity-results-list` you mentioned in your HTML(in the issue) is nowhere to be found in my HTML and that is what causes the script to break on my side. This is a serious issue as I assumed that the HTML would be the same for everyone. Would you mind attaching a screenshot of the LinkedIn page where this is happening? Also, let me know if you'd like to pair-program to solve this issue. I cannot see any solution to this issue while working alone.

**Comment by rajkumaar23** [comment]
@ayushjainrksh 
image


If it's not the same, can we have **your** selector as the **primary** one and **fallback** to what **I** wrote ?

**Comment by ayushjainrksh** [comment]
Ahh, this is a **totally different UI**. So basically, we want to incorporate both. As far as I know, most people see LinkedIn the way I do. This must be a part of either A/B testing or might be because of a local theme on your pc. What do you think?

**Comment by ayushjainrksh** [comment]
To give you an idea

Screenshot from 2020-10-01 19-49-56

**Comment by rajkumaar23** [comment]
ah right @ayushjainrksh !
yours is the standard old UI. yeah probably they’re testing.
so i think i’ll add my selectors as a fallback ?

**Comment by ayushjainrksh** [comment]
Yeah, sounds good. Firstly, it should look for the selectors I've used and in case the script is unable to find the standard selectors, we should use your selectors.

**Comment by rajkumaar23** [comment]
@ayushjainrksh let me know if this works for you!

File: scrape.js [hunk 1]
```diff
@@ -133,8 +134,9 @@ const scrapeLinkedIn = async (data) => {
         let profileLink = profileLinks[employeeUrl];
 
         //Visit activity page
-        await page.goto(profileLink + "detail/recent-activity");
-
+        await page.goto(profileLink + "/detail/recent-activity", {
+          waitUntil: ["domcontentloaded"],
+        });
```
Comment (ayushjainrksh) [inline scrape.js:139]: To be replaced with:
```
await page.goto(profileLink + "/detail/recent-activity", {
          waitUntil: ["domcontentloaded", "networkidle2"],
        });
```

**Review by ayushjainrksh — APPROVED** [review]

**Comment by ayushjainrksh** [comment]
Everything works great for me. I'm gonna add `networkidle2` as well here.

**Comment by ayushjainrksh** [comment]
@rajkumaar23 Thanks for the PR :tada:

**Pull request merged**